OWNERS_ALIASES

```diff
@@ -10,7 +10,7 @@ aliases:
       - stu-gott
       - SchSeba
       - fabiand
-      - codificat
```

Comment:
@mazzystr do you want to move pep to the emeritus section?

Reply:
yes please